supplement.csv

```diff
@@ -1,7 +1,94 @@
 Lemma,Sense,Word Class,Year,Quotation,Source,Note
+愛國者,/,noun,1899,有憂國者，有愛國者。,梁啓超『自由書·憂國與愛國』,
```

Comment:
Is that a `,` in the 1899 edition?

Reply:
By the way, can I indicate that there are more instances of 愛國者 in the same source by adding a + after the quote?

Reply:
> can I indicate that there are more instances of 愛國者 in the same source by adding a + after the quote?

By all means! You can do anything that’s codified in the Conventions.

@RuoyanC

Reply:
> It was a 。 in the 1934 edition of [飲冰室全集](https://commons.wikimedia.org/wiki/File:SSID-13347803_%E9%A3%B2%E5%86%B0%E5%AE%A4%E5%85%A8%E9%9B%86_%E4%B8%8A.pdf). 自由書 was originally written in 1899.

Please add a note reminding others that you didn’t get to check the 1899 original.

This is actually non-trivial: even when an author personally oversaw the production of their own collected edition, there’s no guarantee that what they print there reflects what they printed the first time.

What gets changed may be a punctuation mark, but it could just as likely be a word.